src/components/PersonalBlock.vue

```diff
@@ -0,0 +1,173 @@
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+	name: 'PersonalBlock',
+	components: {},
+
+	data() {
+		return {}
+	},
+
+	props: {
+		fullName: {
+			readonly: true,
+			default: '',
+			type: String,
+		},
+	},
+
+	computed: {
+		firstName(): string {
+			return this.fullName.split(' ')[0];
+		},
+		sAge(): string {
+			const now: number = new Date().getTime();
+			return Math.floor((now - new Date('2021-11-01').getTime()) / 3.15576e+10).toString();
+		}
+	},
+});
+</script>
+
+<template>
+	<details>
+		<summary>
+			<span class="screenreader-only">{{ fullName }} </span>is a full-stack web
+			developer<span class="hide-on-x-small-screen"> at <span
+					class="highlight">Benevity</span></span>.
+		</summary>
+		<article class="details-content">
+			<h2 class="screenreader-only">About {{ firstName }}</h2>
+			<!-- Mmmmmmmm.... Delicious connntenntt... [Homer Simpson voice 🤤] -->
+			<p>
+				At <a href="https://benevity.com/">Benevity</a>, he works with a talented
+				team of software developers who are responsible for dispersing
+				company-seeded funds to users. Previously at Benevity, he worked on Spark;
+				the company's flagship product which is responsible for corporate giving
+				programs all over the globe. Prior to this, he was a Technical Analyst
+				(User Experience Design) with the <a href="https://www2.gov.bc.ca/">Public
+					Service of British Columbia</a> where he created web applications such
+				as <a href="https://studentaidbc.ca/">StudentAidBC</a>.
+			</p>
+			<p>
+				When not programming, Jonathan tends to <a
+					href="/pictures/_MG_3954-Edit.jpg">climb up small boulders in the
```

Comment:
Don't lose your traffic. Keep them there.
```suggestion
					href="/pictures/_MG_3954-Edit.jpg" target="_blank">climb up small boulders in the
```